test/fail_compilation/test17380spec.d

```diff
@@ -1,9 +1,11 @@
 /* REQUIRED_ARGS: -verrors=spec
 TEST_OUTPUT:
 ---
-(spec:1) fail_compilation/test17380spec.d(14): Error: cannot resolve identifier `ThisTypeDoesNotExistAndCrashesTheCompiler`
-(spec:1) fail_compilation/test17380spec.d(14): Error: no property `ThisTypeDoesNotExistAndCrashesTheCompiler` for type `Uint128`
-fail_compilation/test17380spec.d(14): Error: undefined identifier `ThisTypeDoesNotExistAndCrashesTheCompiler`
+(spec:1) fail_compilation/test17380spec.d(16): Error: cannot resolve identifier `ThisTypeDoesNotExistAndCrashesTheCompiler`
+(spec:1) fail_compilation/test17380spec.d(16): Error: no property `ThisTypeDoesNotExistAndCrashesTheCompiler` for type `Uint128`
+(spec:1) fail_compilation/test17380spec.d(16): Error: cannot resolve identifier `ThisTypeDoesNotExistAndCrashesTheCompiler`
+(spec:1) fail_compilation/test17380spec.d(16): Error: no property `ThisTypeDoesNotExistAndCrashesTheCompiler` for type `Int128`
```

Comment:
I don't understand what these speculative errors are all about, and I'm not sure why the changes made affect this test.  Reviewer beware.

Reply:
Speculative are gagged errors. Makes sense to me. That extra error is related to your change (it now checks explicit this.doesnotexist as well as trying to resolve doesnotexist).